Services/BookingService.cs

```diff
@@ -28,4 +28,18 @@ public async Task<List<Booking>> GetMyBookings(Guid userId)
         return bookings;
     }
 
+    public async Task<Booking> SetBookingAsCompleted( string bookingId)
+    {
+        var booking = await _bookingRepository.GetById(bookingId);
+
+        if(booking == null)
```

Comment:
If you add null check in repository, this is not needed anymore as `await _bookingRepository.GetById(bookingId);` will always return Booking object.